hack/checkCreatedAtAnnotation.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Can we keep this file name lowercase, or use dashes instead? The other files in this directory follow this pattern.

Reply:
Ok